demo/dotnet/README.md

```diff
@@ -17,32 +17,27 @@ Porcupine is:
 - scalable. It can detect multiple always-listening voice commands with no added CPU/memory footprint.
 - self-service. Developers can train custom wake phrases using [Picovoice Console](https://picovoice.ai/console/).
 
-## Compatibility
+## Requirements
```

Comment:
My notes on the binding requirements don't apply to this. .NET Core 3.1 is required to run the demo